pages/index.html

```diff
@@ -1,12 +1,46 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Document</title>
+<link href="../styles/style.css" rel="stylesheet" />
 </head>
 <body>
-  
+    <div class="container">
+        <div class="header">
+          <p> <h1>Intesar Siraj</h1></p>
+        </div>
+        <div class="sidebar">
+            <img class="img-side" src="../images/side.jpeg" alt="my image">
+        </div>
+        <div class="content">
+            <p>
```

Comment:
Not everything should be wrapped in a `p` tag. The paragraph tag should be used when putting in a paragraph of text. There generally are not headings inside of paragraphs since headings signify the start of a new section of the page. So many things being inside the `p` tags is causing errors.